PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -934,3 +946,12 @@ public final class DefaultControlsViewController: ContentControlsViewController
         currentUIProps.settingsButtonAction.perform()
     }
 }
+
+extension DefaultControlsViewController: UIGestureRecognizerDelegate {
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
+        if touch.view is UIButton {
```

Comment:
`guard` here?

Reply:
What for? If `touch` doesn't have `view`, `if` statement will be false. 
And, it simply can not have nil in view, because this method will be called after touch was began. And we don't operate with gestures in code, so it will be called only after tap on any view.

Reply:
`return touch.view is UIButton`